backend/remote-state/cos/backend.go

```diff
@@ -0,0 +1,170 @@
+package cos
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform/backend"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common"
+	"github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common/profile"
+	tag "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/tag/v20180813"
```

Comment:
This import looks a bit off -- will investigate, but we use Go modules, and wouldn't want to add code that makes it difficult for us to upgrade to future versions of Go

Reply:
It's fine! Resolving this comment.

Reply:
Unresolving to add a non-blocking comment: If you _can_ update this path to be a major version (https://github.com/golang/go/wiki/Modules#why-must-major-version-numbers-appear-in-import-paths) that would be great, but this is not a blocking comment as other imports in the project have the same issue, which we will have to visit when we upgrade to go1.14.

Reply:
Thanks for your comment, tencentcloud will update it soon.

Reply:
Yes, this work is valueable, but I have to wait for the sdk team updating the sdk before I can update the import paths. I will update it when ready.